### DISCUSSION TITLE
patch_2抢票，无法生成订单，终端显示“票价发生变化”，不知道是不是个别票的问题

### DISCUSSION THREAD

**Discussion by GKDSSS** [mikumifa/biliTickerBuy#421] [提问专区] [answered]
![屏幕截图 2024-08-30 153347](https://github.com/user-attachments/assets/645b4931-e693-4128-9784-96abc8d46aa4)

**Comment by mikumifa** [comment]
给个票id

**Reply by GKDSSS** [reply, marked as answer]
https://show.bilibili.com/platform/detail.html?id=91530&from=pc_ticketlist
就是个别票的问题，这个票降价了。试了别的票可以运行

**Comment by mikumifa** [comment]
降价是怎么回事，他不是一直都是58吗

**Reply by GKDSSS** [reply]
我之间看好像是59.9
